docs/proposals/03-loadbalancer.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Network Loadbalancer
+
+oep-number: 3
+
+creation-date: 2022-10-18
+
+status: implementable
+
+authors:
+
+- "@gehoern"
+reviewers:
+- "@adracus"
+- "@afritzler"
+- "@guvenc"
+- "@MalteJ"
+
+---
+
+# OEP-3: Network Loadbalancer
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+    - [Details](#details)
+- [Proposal](#proposal)
+
+## Summary
+Network service for Load Balancing is needed in every modern infrastructure (e.g. Kubernetes). Load balancing makes services scalable, fault-tolerant and accessible from external users. The load balancer described here is a L3-Loadbalancer (so it implements an IP level load balancing). The load balancer function is a standard network function that can be to be deployed (as an implementation detail) on a platform/node that is capable of running network functions (e.g. dp-service enabled nodes/nic's) 
+
+## Motivation
+The onMetal network is a fully routed network. To run workloads hosted on several nodes (in Kubernetes terms equivalent to cluster IP's) and not only on a single node (node IP's) solutions like ECMP (Equal cost multi pathing) seem to be the natural solution. But ECMP has certain limitations: the amount of nodes it can be used to distribute traffic to and a hash based algorithm to pin the network flows, that changes - based on the amount of targets - its distributed pinning. 
+To avoid those limitations and to have better control over the load balancer behavior an onMetal load balancer service needs to be provided.
+
+### Goals
+- define a user facing API (onMetal api) for the load balancer
+- enable the user to request an L3 Loadbalancer (only IP addresses) with load balancer address (outside facing) and 
```

Comment:
Meld into sth like `Define an API for managing L3 load balancers with publicly available addresses`

Reply:
done